doc/classes/Navigation2D.xml

```diff
@@ -24,6 +26,7 @@
 			<argument index="0" name="to_point" type="Vector2">
 			</argument>
 			<description>
+				Determines the closest navigation point to the point passed and returns the owning [code]Object[/code] of the point's navigation polygon, usually a [code]NavigationPolygonInstance[/code]. If the polygon was added dynamically with [code]navpoly_add()[/code], the owner that was passed in that call is returned (or [code]null[/code] if no owner was passed).
```

Comment:
Other classes should be ref links: [Object] and [NavigationPolygonInstance]
Methods should be linked too: [method navpoly_add]